.github/workflows/publish.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Cache 📦
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```

Comment:
The older version of this action was using node 12 which is no longer supported.